dev-docs/source/AlgorithmProfiler.rst

```diff
@@ -0,0 +1,40 @@
+==============================
+Work flows algorithm profiling
+==============================
+
+.. contents:: Contents
+    :local:
+
+Summary
+^^^^^^^
+
+Due to the need of investigation of algorithms performance issues, the proper method
```

Comment:
I suggest
"This describes how one can profile reduction workflows (CPU and RAM usage) that use one or more algorithms. The profiling tool requires two steps: the first is to build Mantid with a special CMake flag, and the second is to clone the visualization tool which is currently being kept as a separate project. Available for Linux only"